docs/GettingStarted.md

```diff
@@ -369,19 +303,11 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 
 | Key | Type | Default | Description |
 | --- | --- | --- | --- |
+| `service_name` | String | Global service name (default: `faraday`) | Service name for this specific connection object. |
```

Comment:
Love this table! Can we do this for all the arguments for all the integrations above? Consistency would be really helpful in making the documentation easier to read.

Reply:
Yes, I think we should definitely replicate this more structured way of documenting options. I would just leave for the next iteration (see my other comment).